app/views/pages/attend.html.erb

```diff
@@ -1,4 +1,5 @@
 <h1>Attend Public Page</h1>
+<%= render 'shared/topbanner' %>
 <%= render 'what_you_will_learn' %>
 <%= render 'shared/sponsors' %>
 <%= render 'shared/newsletter'%>
```

Comment:
The footer will give a conflict with master, I think. You should end up leaving it as it is and just inserting your partial into the list that is there in the correct position.